pkg/controller/swiftstorage/swiftstorage_controller.go

```diff
@@ -179,12 +245,41 @@ func (r *ReconcileSwiftStorage) createOrUpdateSts(
 		statefulSet.Spec.Template.Spec.SecurityContext.RunAsGroup = &swiftGroupId
 		statefulSet.Spec.Template.Spec.SecurityContext.RunAsUser = &swiftGroupId
 		volumes := r.swiftServicesVolumes(swiftStorage.Name)
+		storageClassName := "local-storage"
+		statefulSet.Spec.VolumeClaimTemplates = []core.PersistentVolumeClaim{
+			{
+				ObjectMeta: meta.ObjectMeta{
+					Name:      "devices-mount-point",
```

Comment:
Devices mount point is something Swift specific? it sounds really generic

Reply:
can't this pv be just called rings or sth :dancers:

Reply:
changed to `storage-device-mount-point`